### ISSUE TITLE
Is it possible to apply this patch to a newer version of the zoom meeting sdk (V. 6.0.10)  for windows.

### ISSUE THREAD

**Issue by dvshah290** [zoom/meetingsdk-win-raw-recording-sample#4]
I'm trying to look at the solution, and the version number that is mentioned in the Readme is not available in the market place anymore. 
Is this patch applicable to newer versions, or is there any updated version of the patch that is available.

**Comment by SardarkhanStella** [comment]
Yes you can patch new SDK version. Just open the patch file and check the line number of this method void CustomizedUIRecordMgr::onRecordPrivilegeChanged and replace that in patch file.
